### PR TITLE
[ MB-14692, MB-14723 ] Duplicating tags at the root stanza

### PR DESCRIPTION
**Pull request by rogeruiz**
## Summary

This fixes the `operation-tag-defined` error.

## Setup to Run Your Code

I ran `make swagger_generate server_generate` on this branch to get all the necessary files checked into the repository. 

As the reviewer, you can run `lint-yaml-with-spectral` from inside this repository and with the `swagger/admin.yaml` and should see that there are only **✖ 104 problems (42 errors, 62 warnings, 0 infos, 0 hints)** which is different from the default branch of **✖ 130 problems (42 errors, 88 warnings, 0 infos, 0 hints)**

### CODE REVIEW

File: swagger-def/admin.yaml [hunk 1]
```diff
@@ -12,6 +12,18 @@ consumes:
...
+  - name: webhook_subscriptions
+  - name: transportation_service_provider_performances
+  - name: admin_users
+  - name: office
+  - name: electronic_order
+  - name: upload
+  - name: organization
+  - name: notification
+  - name: move
+  - name: users
```
Comment (felipe-lee) [inline swagger-def/admin.yaml:26]: **question**: were these the ones you'd mentioned you wanted to sort before merging?

**Review by felipe-lee — COMMENTED** [review]
Lgtm, thanks for fixing these! Just have the question on sorting and then I can approve

**Review by felipe-lee — DISMISSED** [review]
LGTM!

**Comment by rogeruiz** [comment]
Making this a Draft for a bit @felipe-lee because I want to add names and descriptions to the global tags now that I know a bit more about them.

File: swagger-def/admin.yaml [hunk 2]
```diff
@@ -12,6 +12,51 @@ consumes:
   - application/json
 produces:
   - application/json
+tags:
+  - name: admin_users
+    description: Information about Admin UI users
+    externalDocs:
+      url: https://transcom.github.io/mymove-docs/docs/api
```
Comment (rogeruiz) [inline swagger-def/admin.yaml:19]: Lots of repetitive bits here, but I think it makes for a much better developer experience when reading the docs. I also like how there's a space for external documentation to learn more.

That I believe will give us an advantage in how to organize our documentation better across Docusaurus, Confluence, and Swagger.

**Review by kctruss — APPROVED** [review]
💖

**Pull request merged**